Apps/Src/Task_FaultState.c

```diff
@@ -77,7 +81,9 @@ void EnterFaultState() {
     }
 
     // avoid infinite recursive faults, since our CAN Driver relies on the OS to work
-    if ((Fault_BitMap & Fault_OS) == 0) {
+    // also don't call CAN if the watchdog tripped, since CAN won't be initialized
+    /*
+    if ((Fault_BitMap & (Fault_OS | Fault_WDOG)) == 0) {
```

Comment:
If this is all commented, is there a purpose to keeping it around? Is it just in case we need it in the future?

Reply:
We can keep it in if we might use it in the future!